conf/webpack-dist.conf.js

```diff
@@ -5,63 +5,59 @@ const path = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const ExtractTextPlugin = require('extract-text-webpack-plugin');
 const pkg = require('../package.json');
-const autoprefixer = require('autoprefixer');
 
 module.exports = {
-  module: {
-    preLoaders: [
-      {
-        test: /\.js$/,
-        exclude: /node_modules/,
-        loader: 'eslint'
-      }
-    ],
-
-    loaders: [
-      {
-        test: /.json$/,
+    module: {
         loaders: [
-          'json'
+            {
+                test: /\.js$/,
+                exclude: /node_modules/,
+                loader: 'eslint-loader',
+                enforce: 'pre'
+            },
+            {
+                test: /.json$/,
+                loader: 'json-loader'
```

Comment:
json-loader different than json?

Reply:
I'm going to interpret this as asking why the configuration went from

```js
{
    test: /.json$/,
    loaders: [
        'json'
    ]
}
```

to 

```js
{
    test: /.json$/,
    loader: 'json-loader'
}
```

and the answer is that [upgrading webpack](https://webpack.js.org/guides/migrating/#automatic-loader-module-name-extension-removed) required it.